lib/screens/about_page.dart

```diff
@@ -15,8 +22,50 @@ class AboutScreen extends StatelessWidget {
     );
   }
 
+  String getVersionString(Version? newVersion) {
+    var v = version.toString();
+
+    if (newVersion != null) {
+      v += " (neue Version " + newVersion.toString() + " verfügbar)";
+    }
+
+    return v;
+  }
+
+  Future<VersionAndUrl?> getVersionAndUrl() async {
+    var v = await getNewVersion();
+    var url = await newVersionUrl();
+
+    return new VersionAndUrl(v, url);

```

Comment:
Dont use new in Dart, as the compiler will use new or const where it fits
```suggestion
    return VersionAndUrl(v, url);

```